x-pack/plugins/lens/public/app_plugin/shared/edit_on_the_fly/flyout_wrapper.tsx

```diff
@@ -62,18 +62,10 @@ export const FlyoutWrapper = ({
                         values: { lang: language },
                       })}
                   <EuiToolTip
-                    content={
-                      language
-                        ? i18n.translate('xpack.lens.config.experimentalLabelTextBased', {
-                            defaultMessage:
-                              'Technical preview, {lang} currently offers limited configuration options',
-                            values: { lang: language },
-                          })
-                        : i18n.translate('xpack.lens.config.experimentalLabelDataview', {
-                            defaultMessage:
-                              'Technical preview, inline editing currently offers limited configuration options',
-                          })
-                    }
+                    content={i18n.translate('xpack.lens.config.experimentalLabelDataview', {
```

Comment:
nit: you could use the ExperimentalBadge component here and adjust the margins for EuiBetaBadge in some wrapper 
```
<ExperimentalBadge label={i18n.translate('xpack.lens.config.experimentalLabelDataview', {defaultMessage: 'Technical preview, inline editing currently offers limited configuration options',})}/>
```

Reply:
I didnt add new badges, just removed existing ones!

Reply:
Yeah I see that, that's why I marked it as a nit. You've simplified the Badge content and it would be nice to make it reuse the component now when we modify this code, but it's up to you. Feel free to ignore 🤗